string name 
uint64 bytes_sent
uint64 bytes_recv 
uint64 packets_sent 
uint64 packets_recv
uint64 errin 
uint64 errout 
uint64 dropin 
uint64 dropout
bool isup 
string duplex 
uint32 speed 
uint32 mtu 
string flags

